### PR TITLE
Taylor - adding user to existing session

### LINKED ISSUE #212: Taylor - Adding User to Existing Session
Write a function that adds a user to an existing session in a database

### PR DESCRIPTION
**Pull request by TaylorDJones11**
## Description of what this Pull Request does
- Adds User to Existing Session
- Fix bug in authSignup

## Related Tickets & Documents
- Closes #212 

## Screenshots
AddUserToSession
Screenshot 2023-06-18 at 21 10 28

### CODE REVIEW

File: development/src/composables/addUserToSession.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+async function addUserToSession(user, peerId) {
+  try {
+    if (!user || !peerId) {
+      throw new Error("User or session ID is missing");
+    }
+    const sessionRef = doc(appFirestore, "SESSIONS", peerId);
+
+    await updateDoc(sessionRef, {
+      users: arrayUnion(user),
```
Comment (annonymousauthority) [inline development/src/composables/addUserToSession.js:12]: is `user` filed an array?

File: development/src/components/joinSession_Comp.js
Comment (annonymousauthority) [inline development/src/components/joinSession_Comp.js:1]: There's no need for `console.log()` unless during testing

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: development/src/components/joinSession_Comp.js [hunk 2]
```diff
@@ -17,11 +18,26 @@ function JoinSession() {
...
+  const handleJoinSession = async (e) => {
     e.preventDefault();
+
+    const user = {
+      name: "Janet Smith",
+      email: "janetsmith@gmail.com",
+    };
+
+    // Hardcoded session ID
+    const hardcodedSessionId = "76cH5HyEWEhglasWBLl9";
```
Comment (Remi-dee) [inline development/src/components/joinSession_Comp.js:30]: This will be provided from joinSession modal, you can just pass it in as a parameter

**Review by Remi-dee — CHANGES_REQUESTED** [review]

**Review by Remi-dee — DISMISSED** [review]
LGTM

**Review by Remi-dee — APPROVED** [review]

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**